server/sonar-web/src/main/js/apps/project-admin/deletion/Form.js

```diff
@@ -63,7 +63,7 @@ export default class Form extends React.PureComponent {
       .catch(this.stopLoading);
   };
 
-  handleCloseClick = (event: Event) => {
+  handleCloseClick = (event /*: Event */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)